docs/features.rst

```diff
@@ -14,9 +14,7 @@ Configuration, Packaging & Distribution
 
 All configuration can be done in ``setup.cfg`` like changing the description,
 URL, classifiers, installation requirements and so on as defined by setuptools_.
-That means in most cases it is not necessary to tamper with ``setup.py``. In
-fact, if you don't do `editable installs`_ you don't even need it (in the
```

Comment:
I deleted this sentence since most users won't understand it. Right now we have a ``setup.py`` which might change in version 5 but that one could already delete it (if one doesn't need editable installs) is rather an implementation detail that rather confuses users at this point. Also, most developers using PyScaffold will make heavy us of editable installs, right?